### PR TITLE
𝕂 𝕌 𝔹 𝔼 ℝ ℕ 𝔼 𝕋 𝔼 𝕊

### PR DESCRIPTION
**Pull request by DOCtorActoAntohich**
man i love kubernetes

### CODE REVIEW

**Comment by Andrew174194** [comment]
Nice pic 😳👍

File: k8s/README.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+```
+
+`kubectl get pod,svc` yields:
+
+```txt
+NAME                          READY   STATUS    RESTARTS   AGE
+pod/mongus-599fb5f495-v7vnx   1/1     Running   0          0s
+
+NAME                 TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)          AGE
+service/kubernetes   ClusterIP      10.96.0.1       <none>          443/TCP          16y
```
Comment (Andrew174194) [inline k8s/README.md:78]: If don't run Kubernetes for 16 years, don't call me on your party

File: k8s/README.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+## Cleaning up
+
+Similarly to manual work, here are the commands:
+
+```bash
+kubectl delete -f ./config/service.yaml
+kubectl delete -f ./config/deployment.yaml
+```
+
+Then kill `minikube` if you want (I know you *really* want).
```
Comment (Andrew174194) [inline k8s/README.md:157]: truetrueiagree

But to be honest, this lab is pretty OK, it didn't call bouts of hatred for me

Comment (DOCtorActoAntohich) [reply k8s/README.md:157]: kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill kill

File: k8s/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+`kubectl get pod,svc` yields:
+
+```txt
+NAME                          READY   STATUS    RESTARTS   AGE
+pod/mongus-599fb5f495-v7vnx   1/1     Running   0          0s
+
+NAME                 TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)          AGE
+service/kubernetes   ClusterIP      10.96.0.1       <none>          443/TCP          16y
+service/mongus       LoadBalancer   10.104.64.115   10.104.64.115   8000:32241/TCP   0s
+```
```
Comment (molberte) [inline k8s/README.md:80]: amazing instructions, doc!!

File: k8s/README.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+`kubectl get pods,svc` yields:
+
+```txt
+NAME                                  READY   STATUS    RESTARTS   AGE
+pod/make-your-time-796d54b48d-5zl4l   1/1     Running   0          0s
+pod/make-your-time-796d54b48d-77p8p   1/1     Running   0          0s
+pod/make-your-time-796d54b48d-bpd6v   1/1     Running   0          0s
+
+NAME                     TYPE           CLUSTER-IP     EXTERNAL-IP   PORT(S)          AGE
+service/kubernetes       ClusterIP      10.96.0.1      <none>        443/TCP          1.7sg
```
Comment (ntdesmond) [inline k8s/README.md:122]: but 1.7sg is much less than 16 years... did you restart the service?

**Review by ntdesmond — COMMENTED** [review]
forgor

File: k8s/README.md (hunk 4 above)
Comment (DOCtorActoAntohich) [reply k8s/README.md:122]: yeah sorry

**Pull request merged**